dev/test/specs/starter-test.js

```diff
@@ -97,6 +97,42 @@ function loadSeleniumUtils() {
   });
 }
 
+function getFooterPath() {
+  return ['arc-footer', 'x-toast[app-footer]'];
+}
+
+function initTestWithNewArc() {
+  // TODO(smalls) should we create a user on the fly?
+  // note - baseUrl (currently specified on the command line) must end in a
+  // trailing '/', and this must not begin with a preceding '/'.
+  browser.url(`apps/web/?user=-L-YGQo_7f3izwPg6RBn`);
+
+  assert.equal('Arcs', browser.getTitle());
+
+  // create a new arc, switch to that tab (toggling back to the first tab to
+  // reset the webdriver window state).
+  createNewArc();
+
+  // wait for the page to load a bit, init the test harness for this page
+  browser.waitForVisible('<app-main>');
+  browser.waitForVisible('<footer>');
+  loadSeleniumUtils();
+
+  // check out some basic structure relative to the app footer
+  const footerPath = getFooterPath(); //['arc-footer', 'x-toast[app-footer]'];
+  assert.ok(pierceShadowsSingle(footerPath.slice(0, 1)).value);
+  assert.ok(pierceShadowsSingle(footerPath).value);
+}
+
+function createNewArc() {
+  // create a new arc, switch to that tab (toggling back to the first tab to
+  // reset the webdriver window state).
+  browser.waitForVisible('div[title="New Arc"]');
```

Comment:
Now that we have multiple tests, there's the chance that we might finish through one of the tests to browser.close() and we'd get in a weird state here.

Maybe add a precondition that there's only 1 open window at the start of this method?

Reply:
done (although I'm not sure this is absolutely correct - we might want multiple arcs and hence multiple windows in the same test case later.. but it's good for now).